.github/workflows/nix-flake-lock.yml

```diff
@@ -23,9 +23,38 @@ jobs:
         # https://github.com/DeterminateSystems/update-flake-lock/releases/tag/v19
         uses: DeterminateSystems/update-flake-lock@dec3bc3c9b11c3b9d547f47dfb579b91a6051603
         with:
-          git-author-name: 'ockam[bot]'
-          git-author-email: 'ockam[bot]@users.noreply.ockam.io'
-          git-committer-name: 'ockam[bot]'
-          git-committer-email: 'ockam[bot]@users.noreply.ockam.io'
+          commit-msg: "build: update tools/nix/flake.lock"
+          git-author-name: '${{ secrets.GPG_USER_NAME }}'
```

Comment:
@metaclips do we know yet what name of Environment should be filled in for correct access to this information?

Reply:
I informed @mrinalwadhwa on this